app/models/shelter.rb

```diff
@@ -4,6 +4,7 @@ class Shelter < ApplicationRecord
   validates :city, presence: true
 
   has_many :pets, dependent: :destroy
+  has_many :applications, through: :pets
```

Comment:
Excellent work adding the additional association so that you can join shelters directly to applications